metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -126,7 +132,7 @@ final class Doctor(
   def check(serverName: String, serverVersion: String): Unit = {
     bspServerName = Some(serverName)
     bspServerVersion = Some(serverVersion)
-    val summary = problemSummary
+    val summary = problemResolver.problemSummary(allTargets())
```

Comment:
Mentioned this at the definition site, but I think this is a bit misleading since `summary` here isn't really a summary, but just a message about issues. What do you think of actually including all of those messages if there are more than one here?

Reply:
I renamed it to `problemMessage`, but I think we should stay with a single message, since it avoids spamming the user and the `More information` option should be visible enough for people to easily check what is going on.

Reply:
yea sounds good 👍🏼